core/php8.0Action/runner.php

```diff
@@ -85,8 +86,8 @@
     }
 
     // ensure that the sentinels will be on their own lines
-    file_put_contents('php://stderr', "\n");
-    file_put_contents('php://stdout', "\n");
+    file_put_contents('php://stderr', "\n" . $sentinel);
+    file_put_contents('php://stdout', "\n" . $sentinel);
```

Comment:
Note: I believe this newline is causing us to write "unnecessary" logs, but I wanted to retain the existing behavior in this PR. If we find that we want to change the behavior, we should do so separately.

Reply:
it covers a corner case? if function logs without a new line, the sentinel gets mangled as part of that log line (this is likely the case everywhere else).